src/calendar/calendar-class.js

```diff
@@ -398,6 +400,8 @@ class Calendar {
         - None, but does modify the calendar table HTML
      */
     _populateCalendarMonth(calendarDateList) {
+        // const myDate = new Date("2024-06-02T07:00:00.000Z");
+        const myDate = new Date(2024, 5, 2);
```

Comment:
why do we hardcode here? Maybe there's context, I'm just not too familar w the class

Reply:
Leftovers from debugging, just deleted it.